mat64/svd.go

```diff
@@ -6,474 +6,182 @@
 package mat64
 
 import (
-	"math"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/lapack"
+	"github.com/gonum/lapack/lapack64"
+	"github.com/gonum/matrix"
 )
 
-type SVDFactors struct {
-	U     *Dense
-	Sigma []float64
-	V     *Dense
-	m, n  int
+// SVDKind specifies the treatment of singular vectors during the
+// factorization.
+// NOTE: Should these be in matrix?
```

Comment:
Sorry, I should be more clear. Do you see any utility for a cmat128 SVD? If there is, then these should go in matrix. If there isn't then maybe they should go in matrix.